map/docs/define-map.md

```diff
@@ -296,4 +296,23 @@ myType.myProp = "value"; // no error thrown
 myType.otherProp = "value" // throws Error!
 ```
 
+Sealed objects are extendable and the extended objects can be unsealed:
```

Comment:
This is a bit too much information for the overview.  Lets keep this in `seal`'s docs.